Source/WEB_ROOT/Inventory/js/assets.js

```diff
@@ -12,8 +12,22 @@ function CheckOutDateBtn()
 $j(document).ready(function(){
     $j('#searchBoxBtn').click(function(){
 	    //The Search button was clicked
-		console.log("Search was clicked : " + $j('#searchBox').val());
+		//console.log("Search was clicked : " + $j('#searchBox').val());
 		MakeRequest($j('#searchBox').val());
 
 	});
-});
\ No newline at end of file
+});
+
+

```

Comment:
I think you were on the right track John. It simply doesn't know what "input" is. If you were to steal the line above with $j('#searchBoxBtn') and slap .keypress on the end of that with the rest of your code it would likely do the trick. It's effectively saying hey John, what html control and i listening for an enter key press on? 

The top answer here gives a pretty good example: https://stackoverflow.com/questions/18160342/jquery-how-to-trigger-click-event-on-pressing-enter-key

Reply:
I tried .keypress and .onkeypress,  both don't work.